### PR TITLE
Use . instead of # in describe

### PR DESCRIPTION
**Pull request by takiy33**
Thank you great faker.
I replace `#` with `.` because it is class method

### CODE REVIEW

**Comment by sue445** [comment]
@takiy33 https://travis-ci.org/sue445/faker-precure/builds/18042494 is failed, but I fixed this problem fix at #6 

Please `rebase` or `merge` current master 🙇

**Comment by takiy33** [comment]
I have rebased but, CI if failed. I will see later.
Any way, Thank you.

**Comment by takiy33** [comment]
CI is red because `Faker::Precure.transform_message` has nil.
I check with the following code.

```ruby
[8] pry(main)> Precure.all_stars.select{|e| e.transform_message.nil? }.count
=> 3
```

File: lib/faker/precure.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def self.precure_name
 
     # @return [String]
     def self.transform_message
-      sample_girl.transform_message
+      ::Precure.all_stars.map(&:transform_message).compact.sample
```
Comment (takiy33) [inline lib/faker/precure.rb]: I am not confident of this change. If you have good idea, Please teach me 🙏

Comment (sue445) [inline lib/faker/precure.rb]: LGTM 👍

**Comment by sue445** [comment]
thx!

**Pull request merged**